packages/host/app/components/operator-mode/card-schema-editor.gts

```diff
@@ -103,28 +104,45 @@ export default class CardSchemaEditor extends Component<Signature> {
         height: 20px;
         width: 20px;
       }
+
+      .header {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+      }
+
+      .total-fields {
+        font: 600 var(--boxel-font);
+      }
     </style>
 
     <div
       class='schema-editor-container'
       data-test-card-schema={{@cardType.displayName}}
     >
-      <div class='pill'>
-        <div class='realm-icon'>
-          <RealmInfoProvider @fileURL={{@cardType.module}}>
-            <:ready as |realmInfo|>
-              <img
-                src={{realmInfo.iconURL}}
-                alt='Realm icon'
-                data-test-realm-icon-url={{realmInfo.iconURL}}
-              />
-            </:ready>
-          </RealmInfoProvider>
+      <div class='header'>
+        <div class='pill'>
+          <div class='realm-icon'>
+            <RealmInfoProvider @fileURL={{@cardType.module}}>
+              <:ready as |realmInfo|>
+                <img
+                  src={{realmInfo.iconURL}}
+                  alt='Realm icon'
+                  data-test-realm-icon-url={{realmInfo.iconURL}}
+                />
+              </:ready>
+            </RealmInfoProvider>
+          </div>
+          <div>
+            <span>
+              {{@cardType.displayName}}
+            </span>
+          </div>
         </div>
-        <div>
-          <span>
-            {{@cardType.displayName}}
-          </span>
+        <div data-test-total-fields>
+          <span class='total-fields'>+
+            {{this.totalOwnField}}</span>
+          {{if (gt this.totalOwnField 1) 'Fields' 'Field'}}
```

Comment:
Is there a 0 scenario?

Reply:
English weird. 0 fields rather than 0 field. I wonder if`pluralize` or `getPlural` function might solve this.

Reply:
Good idea! I'll try to use it.

Reply:
Personally in the case of 0 fields, i think it sounds a lot better in English to simply say "No Fields"

Reply:
I decided to implement @habdelra 's suggestion since it sounds better. [73b4a74](https://github.com/cardstack/boxel/pull/701/commits/73b4a74f17f78be070684b59e111e232827807f6)

<img width="364" alt="Screenshot 2023-10-02 at 21 29 50" src="https://github.com/cardstack/boxel/assets/12637010/22baf10b-0c73-4bf1-9f50-c0d5a5425188">